packages/integration-tests/package.json

```diff
@@ -26,7 +26,7 @@
     "@jest/types": "^29.1.2",
     "@logto/connector-kit": "workspace:^3.0.0",
     "@logto/core-kit": "workspace:^",
-    "@logto/js": "^4.1.1",
+    "@logto/js": "^4.1.4",
     "@logto/node": "^2.4.7",
```

Comment:
Should we also bump `@logto/node`? Saw the latest version is `2.5.4`

Reply:
theoretically we should - we can bump it later